pkg/coderefs/coderefs.go

```diff
@@ -278,12 +286,21 @@ func generateReferences(flags []string, searchResult [][]string, ctxLines int, d
 	return references
 }
 
-func findReferencedFlags(ref string, flags []string, delims string) []string {
-	ret := []string{}
-	for _, flag := range flags {
-		matcher := regexp.MustCompile(fmt.Sprintf("[%s]%s[%s]", delims, flag, delims))
+func findReferencedFlags(ref string, aliases map[string][]string, delims string) map[string][]string {
+	ret := make(map[string][]string, len(aliases))
+	for key, flagAliases := range aliases {
+		matcher := regexp.MustCompile(fmt.Sprintf("[%s]%s[%s]", delims, regexp.QuoteMeta(key), delims))
```

Comment:
were we never handling flags with dots correctly before? since we weren't escaping the dot

Reply:
exactly - i included this as a bug fix in the [changelog pr](https://github.com/launchdarkly/ld-find-code-refs/pull/111)